cypress/integration/BasicTermPageSpanish/BasicTermPageSpanish.js

```diff
@@ -0,0 +1,12 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Given('{string} is set to {string}', (key, param) => {
```

Comment:
Can you move this to common so that it is available everywhere.

Reply:
moved to common

Reply:
Also added HealthProfessionals from genetics dictionary tests